recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -39,9 +60,9 @@ do_compile() {
     env -i HOME="$HOME" LC_CTYPE="${LC_ALL:-${LC_CTYPE:-$LANG}}" USER="$USER" \
            PATH="${USRBINPATH_NATIVE}:${PATH}" \
            OUT_DIR_COMMON_BASE="${ANDROID_OUT_DIR_COMMON_BASE}" \
-           DDK_KM_PREBUILT_MODULE="${DDK_KM_PREBUILT_MODULE}" \
-           TARGET_PREBUILT_KERNEL="${TARGET_PREBUILT_KERNEL}" \
-           DDK_UM_PREBUILDS="${DDK_UM_PREBUILDS}" \
+           ${@d.getVar('PARAM_DDK_KM_PREBUILDS_MODULE', True)} \
```

Comment:
`${PARAM_DDK_KM_PREBUILDS_MODULE}` will work as well. You don't need to use this monstrosity with ` ${@d.getVar('')}`

Reply:
it is clear that it works, but , my vision , if we use setVar it is reasonable to use getVar as well , to have the same approach set/get